setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="tap_bigquery",
-    version="0.1.4",
+    version="0.1.5",
```

Comment:
Can you mark version="0.2.0" since this is a breaking change for existing downstream processes? Thx.

Reply:
Done